_posts/2015/2016-07-20-fullstackcon-2016-retrospective.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "ePages goes FullStack JavaScript conference"
+date: "2016-07-19 14:00:00"
+image: blog-header/javascript.png
+categories: events
+authors: ["Paolo"]
+---
+
+The Brexit is coming, the pound sterling is low, and my fellow colleague Erik and me took the opportunity to visit a conference in London ~~before we'll need a visa for that~~. It was called [_FullStack - the conference on JavaScript, Node & Internet of Things_][fullstack-website] and featured, among others, speakers from Google, Microsoft, and Facebook.
```

Comment:
Erik & me -> Erik and I

In this case you use I not me because the sentence reads 'me took the opportunity.'   Quick test:  eliminate the others and then use I or me appropriately.